tutorials/llm/llama/domain-adaptive-pretraining/README.md

```diff
@@ -0,0 +1,16 @@
+# ChipNeMo - Custom tokenization + Domain Adaptive Pre-training on Llama 2 70b with NeMo Framework
```

Comment:
Should we
1. clear the notebook output?
2. clear the unnecessary comments in the code?

Reply:
I have cleaned the unnecessary notebook output but we have decided to keep some of the outputs in the notebook that are informative, and useful for the reader in understanding what to expect when they run those code blocks.

Also cleared the unnecessary comments in the code.